truss-chains/truss_chains/utils.py

```diff
@@ -11,11 +12,12 @@
 import textwrap
 import threading
 import traceback
-from typing import Any, Iterable, Iterator, NoReturn, Type, TypeVar, Union
+from typing import Any, Iterable, Iterator, Mapping, NoReturn, Type, TypeVar, Union
 
 import fastapi
 import httpx
 import pydantic
+from truss.templates.shared import dynamic_config_resolver
```

Comment:
Not in scope of this PR, but the fact we have this "templates" dir and how stuff inside the truss repo depends on each uter just makes me wanna jump up the walls...